pulp_galaxy_ng/Containerfile

```diff
@@ -8,6 +8,6 @@ RUN pip3 install --upgrade \
   galaxy-ng${GALAXY_NG_VERSION} && \
   rm -rf /root/.cache/pip
 
-RUN ln /usr/local/lib/python3.8/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_ansible.conf
```

Comment:
this is the only concern I have, we'll stop to test pulp on python 3.8

Reply:
Yeah, there is no way around this. Unless we want to maintain 2 different images, which is not ideal.

I will close this PR but not delete my branch. We will implement this later.